config/development.json

```diff
@@ -112,7 +112,7 @@
 		"reader/list-management": true,
 		"reader/recommendations/posts": true,
 		"reader/related-posts": true,
-		"reader/refresh-2016-07": true,
+		"reader/refresh-2016-07": false,
```

Comment:
made this false by default because otherwise folks are going to have a bad time with local dev and reader